src/Tizen.NUI.Components/Style/ScrollbarStyle.cs

```diff
@@ -54,7 +54,7 @@ public class ScrollbarStyle : ViewStyle
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static readonly BindableProperty TrackColorProperty = BindableProperty.Create(nameof(TrackColor), typeof(Color), typeof(ScrollbarStyle), null, propertyChanged: (bindable, oldValue, newValue) =>
         {
-            ((ScrollbarStyle)bindable).trackColor = (Color)newValue;
+            ((ScrollbarStyle)bindable).trackColor = new Color((Color)newValue);
```

Comment:
These local variables are used for `BindableObject.GetValue()`.

#### Sample Code :
```cs
ScrollbarStyle scrollbarStyle = new ScrollbarStyle();
scrollbarStyle.SetValue(ScrollbarStyle.TrackPaddingProperty, new Extents(10, 10, 10, 10));
Extents extents_get = (Extents)scrollbarStyle.GetValue(ScrollbarStyle.TrackPaddingProperty);
```

#### BackTrace : 
```cs
>	Tizen.NUI.Components.ScrollbarStyle.<>c.<.cctor>b__12_9() Line 83	C# // defaultValueCreator of TrackPaddingProperty
 	Tizen.NUI.Binding.BindableObject.CreateAndAddContext() Line 720	C#
 	Tizen.NUI.Binding.BindableObject.GetOrCreateContext() Line 752	C#
 	Tizen.NUI.Binding.BindableObject.GetValue() Line 149	C#
```

 ---

`BindableObject.GetValue()` and `BindableObject.SetValue()` are different from `Xamarin.Forms`.

If the BindableObject is not created by xaml, it only invoke `PropertyChanged` callback.
#### NUI : 


        public void SetValue(BindableProperty property, object value)
        {
            if (true == isCreateByXaml)
            {
                SetValue(property, value, false, true);
            }
            else
            {
                property.PropertyChanged?.Invoke(this, null, value);

                OnPropertyChanged(property.PropertyName);
            }
        }

Because `BindablePropertyContext` is always null, `defaultValueCreator` is always called in `BindableObject.GetValue()`.

---

If the BindableObject is created by xaml, `Context.Value` will be re-assigned by `DefaultValueCreator`.
#### NUI : 

        BindablePropertyContext GetOrCreateContext(BindableProperty property)
        {
            BindablePropertyContext context = GetContext(property);
            if (context == null)
            {
                context = CreateAndAddContext(property);
            }
            else if (property.DefaultValueCreator != null )
            {
                context.Value = property.DefaultValueCreator(this); //Update Value from dali
            }//added by xb.teng

            return context;
        }